packages/slate/src/utils/memoize.js

```diff
@@ -79,12 +82,26 @@ function memoize(object, properties, options = {}) {
         if (CACHE_KEY !== this.__cache_key) {
           this.__cache_key = CACHE_KEY
           this.__cache = new Map() // eslint-disable-line no-undef,no-restricted-globals
+          this.__cache_no_args = {}
         }
       }
 
       if (!this.__cache) {
         this.__cache = new Map() // eslint-disable-line no-undef,no-restricted-globals
       }
+      if (!this.__cache_no_args) {
+        this.__cache_no_args = {}
+      }
+
+      // The default parameter of text.getLeaves would have an arg with fn.length===0
+      const takesArguments = original.length !== 0 || args.length !== 0
+      if (IS_DEV && takesArguments) {
+        if (original.length > args.length) {
+          throw new Error(
```

Comment:
Can you remove this error? I'm not sure why we need it, but it seems weird for this part of the code to enforce this restriction.

Reply:
Yes, I remove it in the recent commit.